### PR TITLE
Replace git functions by fs ones in Project composable

### PR DESCRIPTION
**Pull request by k-y0u**
Replace git function by fs function in Project composable to get the local result instead of the remote result.

### CODE REVIEW

File: src/composables/Project.js [hunk 1]
```diff
@@ -120,17 +122,50 @@ export async function fetchGit(project) {
   return GitEvent.FetchEvent.next();
 }
 
+async function isDirectory(path) {
```
Comment (Zorin95670) [inline src/composables/Project.js:125]: Missing documentation...

File: src/composables/Project.js [hunk 2]
```diff
@@ -120,17 +122,50 @@ export async function fetchGit(project) {
...
+async function readDir(path) {
+  return new Promise((resolve) => {
+    fs.readdir(
+      path,
+      (e, rv) => resolve(rv),
+    );
+  });
+}
+
+async function getFiles(files, projectId, filename) {
```
Comment (Zorin95670) [inline src/composables/Project.js:144]: if I understand function, the fonction would be rename into setFiles, because we populated the files array.

**Review by Zorin95670 — CHANGES_REQUESTED** [review]

File: src/composables/Project.js (hunk 1 above)
Comment (k-y0u) [reply src/composables/Project.js:130]: Done

File: src/composables/Project.js (hunk 2 above)
Comment (k-y0u) [reply src/composables/Project.js]: Done

File: src/composables/Project.js [hunk 3]
```diff
@@ -120,17 +122,66 @@ export async function fetchGit(project) {
...
+      path,
+      (e, rv) => resolve(rv),
+    );
+  });
+  return stat.isDirectory();
+}
+
+/**
+ * Get the list file/directory found in path location.
+ * @param {Sting} path Path to check.
```
Comment (Zorin95670) [inline src/composables/Project.js:142]: giphy

**Review by Zorin95670 — CHANGES_REQUESTED** [review]

**Review by Zorin95670 — DISMISSED** [review]

File: src/composables/Project.js [hunk 4]
```diff
@@ -120,17 +122,66 @@ export async function fetchGit(project) {
   return GitEvent.FetchEvent.next();
 }
 
+/**
+ * Check if path is directory or not.
+ * @param {String} path - Path to check.
+ * @returns Promise with boolean on success otherwise an error.
```
Comment (Zorin95670) [inline src/composables/Project.js:128]: `@return {Promise<Boolean>} Promise with boolean on success otherwise an error.`

**Review by Zorin95670 — CHANGES_REQUESTED** [review]

**Review by Zorin95670 — APPROVED** [review]

**Pull request merged**